src/web/server/app/tests/controllers/auth.test.ts

```diff
@@ -113,14 +102,95 @@ describe('auth', () => {
       const res = mockRes();
       jest
         .spyOn(authServices, 'checkUserExists')
-        .mockImplementationOnce((userId: string) =>
-          Promise.resolve({ rows: [userId] } as any)
-        );
+        .mockImplementationOnce((userId: string) => Promise.resolve(true));
       await auth.register(req, res);
       expect(res.status).toBeCalledWith(400);
     });
   });
 
+  describe('approveRegistration', () => {
+    const mockReq: any = (username: string) => {
+      return {
+        body: { username },
+      };
+    };
+    const mockRes: any = () => {
+      const res = {
+        status: jest.fn(),
+        send: jest.fn(),
+      };
+      res.status = jest.fn().mockReturnValue(res);
+      res.send = jest.fn().mockReturnValue(res);
+      return res;
+    };
+    jest
+      .spyOn(authServices, 'approveRegistration')
+      .mockImplementation(() => Promise.resolve());
```

Comment:
**()** => Promise.resolve() adding the parameters of the mocked function with it's types might it clearer to read